sklearn/ensemble/voting_classifier.py

```diff
@@ -0,0 +1,189 @@
+"""
+Soft Voting/Majority Rule classifier.
+
+This module contains a Soft Voting/Majority Rule classifier for
+classification estimators.
+
+"""
+
+# Authors: Sebastian Raschka <se.raschka@gmail.com>
+#
+# Licence: BSD 3 clause
+
+from ..base import BaseEstimator
+from ..base import ClassifierMixin
+from ..base import TransformerMixin
+from ..preprocessing import LabelEncoder
+
+import numpy as np
+import operator
+
+
+class VotingClassifier(BaseEstimator, ClassifierMixin, TransformerMixin):
+    
+    """ Soft Voting/Majority Rule classifier for classification estimators.
+
+    Parameters
+    ----------
+    estimators : array-like, shape = [n_classifiers]
+      A list of estimators for classification.
+
+    voting : str, {'hard', 'soft'} (default='hard')
+      If 'hard', uses predicted class labels for majority rule voting.
+      Else if 'soft', predicts the class label based on the argmax of
+      the sums of the predicted probalities.
```

Comment:
I would add that `'soft'` is recommended if all the base `estimators` are expected to yield well calibrated probabilities. Once the the calibration PR #1176 is merged we should add a `See also` section to `CalibratedClassifierCV` at the end of the this docstring.